src/ResourceManager/CognitiveServices/CognitiveServices.Test/Commands.Management.CognitiveServices.Test.csproj

```diff
@@ -106,5 +104,23 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.CognitiveServices, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.CognitiveServices.4.1.0\lib\net452\Microsoft.Azure.Management.CognitiveServices.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@yangyuan you can remove the extra references (`Microsoft.Rest.ClientRuntime`, `Microsoft.Rest.ClientRuntime.Azure`, `Newtonsoft.Json` and `System.Runtime`) that were added in this file -- these are already being referenced in a separate `.targets` file

Reply:
Done